MapboxNavigation-SPM.xcodeproj/project.pbxproj

```diff
@@ -934,7 +934,7 @@
 				CLANG_WARN__DUPLICATE_METHOD_MATCH = YES;
 				"CODE_SIGN_IDENTITY[sdk=iphoneos*]" = "iPhone Developer";
 				COPY_PHASE_STRIP = NO;
-				CURRENT_PROJECT_VERSION = 58;
```

Comment:
I wonder why `MapboxNavigation-SPM.xcodeproj` previosly had version `58`, but `MapboxNavigation.xcodeproj` `59`. Hopefully that's okay to bump by two versions here.

`update-version.sh` also no longer works for two xcprojects. Created ticket here: https://github.com/mapbox/mapbox-navigation-ios/issues/3143.

Reply:
`MapboxNavigation-SPM.xcodeproj` doesn't even need a version actually.